### PR TITLE
align with ShardedCounter for ease of reading

### PR DESCRIPTION
**Pull request by honnix**

### CODE REVIEW

File: styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java [hunk 1]
```diff
@@ -487,16 +487,13 @@ void updateShards(final Storage storage,
       final Long usage = entity.getValue();
       LOG.info("Syncing {} -> {}", resource, usage);
       try {
-        int result = (int) (usage / NUM_SHARDS);
```
Comment (danielnorberg) [inline styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: Does this code belong in `StyxScheduler.java`? Can it be moved to `ShardedCounter.java` instead?

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: That’s also weird. Maybe move to Storage?

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: Better address this in #489 if we want to.

Comment (honnix) [reply styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java]: OK to get it merged for the scope of this PR?

**Comment by NatashaL** [comment]
👍

**Pull request merged**